<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1161</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 10pt &quot;Arial&quot;;
</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>551</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255); 

</string>
      </property>
      <property name="title">
       <string>Acceuil</string>
      </property>
      <widget class="QPushButton" name="pushButton_17">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>70</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(67, 189, 226);
font: 75 10pt &quot;Arial&quot;;
  border-radius: 6px;
  border: none;
color:rgb(255, 255, 255);

</string>
       </property>
       <property name="text">
        <string>Ajouter un bureau</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_18">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>140</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(67, 189, 226);
font: 75 10pt &quot;Arial&quot;;
  border-radius: 6px;
  border: none;
color:rgb(255, 255, 255);


</string>
       </property>
       <property name="text">
        <string>Modifier un bureau</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_19">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>210</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(67, 189, 226);
font: 75 10pt &quot;Arial&quot;;
  border-radius: 6px;
  border: none;
color:rgb(255, 255, 255);

</string>
       </property>
       <property name="text">
        <string>Supprimer un bureau</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_38">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(67, 189, 226);
font: 75 10pt &quot;Arial&quot;;
  border-radius: 6px;
  border: none;
color:rgb(255, 255, 255);


</string>
       </property>
       <property name="text">
        <string>Chercher un bureau</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_41">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(67, 189, 226);
font: 75 10pt &quot;Arial&quot;;
  border-radius: 6px;
  border: none;
color:rgb(255, 255, 255);

</string>
       </property>
       <property name="text">
        <string>Statistiques</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic_Chercher">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic_Ajouter">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic_Modifier">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>140</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic_Supprimer">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>210</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic_Statistiques">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_42">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>210</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(67, 189, 226);
font: 75 10pt &quot;Arial&quot;;
  border-radius: 6px;
  border: none;
color:rgb(255, 255, 255);

</string>
       </property>
       <property name="text">
        <string>QR Code</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic_QR">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic_Acceuil">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>-10</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QGroupBox" name="groupBox_21">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>1171</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="title">
       <string>Bureaux</string>
      </property>
      <widget class="QLabel" name="label_76">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>12</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 100 10pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Trier par :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_77">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Chercher par :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic_Bureau">
       <property name="geometry">
        <rect>
         <x>1100</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>70</y>
         <width>351</width>
         <height>211</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255);</string>
       </property>
       <property name="title">
        <string>Statistiques Bureaux</string>
       </property>
       <widget class="QComboBox" name="comboBox_15">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(67, 189, 226);
border-radius: 10px; 
padding: 0 8px; 
color:rgb(255, 255, 255);
</string>
        </property>
        <item>
         <property name="text">
          <string>Bureau non équipé</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bureau équipé</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_pic_Statistiques1_2">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>-10</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>261</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_13">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>280</y>
         <width>281</width>
         <height>241</height>
        </rect>
       </property>
       <property name="title">
        <string>QR Code</string>
       </property>
       <widget class="QComboBox" name="comboBox_9">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>30</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color: rgb(67, 189, 226);
border-radius: 10px; 
padding: 0 8px; 
color:rgb(255, 255, 255);
</string>
        </property>
        <item>
         <property name="text">
          <string>Bureau 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bureau 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bureau 3</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_pic_QR1_2">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>-10</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>191</width>
          <height>141</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QTableView" name="TabBureau">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>771</width>
         <height>201</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>291</width>
         <height>231</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLineEdit" name="LENum">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 255, 255); 
border: 2px solid gray; 
border-radius: 10px; 
padding: 0 8px; 
selection-background-color: darkgray; 
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Numéro bureau</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LETypeBureau">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>140</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 255, 255); 
border: 2px solid gray; 
border-radius: 10px; 
padding: 0 8px; 
selection-background-color: darkgray; 
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Type Bureau</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Disponibilité</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Capacité MAX</string>
        </property>
       </widget>
       <widget class="QComboBox" name="ComboCapacitemax">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>60</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(232, 232, 232);
border-radius: 10px; 
padding: 0 8px; </string>
        </property>
        <item>
         <property name="text">
          <string>2 Personnes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3 Personnes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4 Personnes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 Personnes</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>180</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 122, 34);
  border-radius: 6px;
  border: none;
color:rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Ajouter un bureau</string>
        </property>
       </widget>
       <widget class="QComboBox" name="ComboDispo">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(232, 232, 232);
border-radius: 10px; 
padding: 0 8px; </string>
        </property>
        <item>
         <property name="text">
          <string>Non Disponible</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disponible</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>1100</x>
         <y>440</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Acceuil </string>
       </property>
      </widget>
      <widget class="QRadioButton" name="trieNum">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Num</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="trieType">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>30</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="trieCapacite">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Capacité</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="trieDisponibilite">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>30</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Disponibilité</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="BaRecherche">
       <property name="geometry">
        <rect>
         <x>740</x>
         <y>30</y>
         <width>331</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255); 
border: 2px solid gray; 
border-radius: 10px; 
padding: 0 8px; 
selection-background-color: darkgray; 
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>290</y>
         <width>281</width>
         <height>231</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pb_modif">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 122, 34);
  border-radius: 6px;
  border: none;
color:rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Modifier un bureau</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>231</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom du bureau à supprimer :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supp">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>170</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 122, 34);
  border-radius: 6px;
  border: none;
color:rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Supprimer un bureau</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LENumsupp">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>140</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 255, 255); 
border: 2px solid gray; 
border-radius: 10px; 
padding: 0 8px; 
selection-background-color: darkgray; 
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>290</y>
         <width>181</width>
         <height>231</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_pic_PDF">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Exporter PDF:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modif_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(67, 189, 226);
  border-radius: 6px;
  border: none;
color:rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_pic_Acceuil_2">
       <property name="geometry">
        <rect>
         <x>1110</x>
         <y>460</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_pic_Bureau</zorder>
      <zorder>label_76</zorder>
      <zorder>label_77</zorder>
      <zorder>groupBox_11</zorder>
      <zorder>groupBox_13</zorder>
      <zorder>TabBureau</zorder>
      <zorder>groupBox</zorder>
      <zorder>label_6</zorder>
      <zorder>trieNum</zorder>
      <zorder>trieType</zorder>
      <zorder>trieCapacite</zorder>
      <zorder>trieDisponibilite</zorder>
      <zorder>BaRecherche</zorder>
      <zorder>groupBox_2</zorder>
      <zorder>groupBox_3</zorder>
      <zorder>label_pic_Acceuil_2</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1176</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
